cmd/helm/search_hub.go

```diff
@@ -91,11 +92,17 @@ func (o *searchHubOptions) run(out io.Writer, args []string) error {
 	return o.outputFormat.Write(out, newHubSearchWriter(results, o.searchEndpoint, o.maxColWidth))
 }
 
+type hubChartRepo struct {
+	URL  string `json:"url"`
+	Name string `json:"name"`
```

Comment:
I don't see this being used anywhere. Can we remove this field?

Perhaps this struct should be collapsed into `hubChartElement` as one field called `Repository` of type `string` rather than of type `hubChartRepo`.

Reply:
If you mean replace both `URL` and `Name` in `hubChartElement` with just one `Repository` which contains the URL (as string), I can do it.
On the other hand,
I used the structure to have "nicer" output in case of JSON, YAML, see below (output is cut):

```json
  {
    "url": "https://artifacthub.io/packages/helm/bitnami/phpmyadmin",
    "version": "8.1.0",
    "app_version": "5.0.4",
    "description": "phpMyAdmin is an mysql administration frontend",
    "repository": {
      "url": "https://charts.bitnami.com/bitnami",
      "name": "bitnami"
    }
  }
```

Here, I think, having repository: {url: "path", name: "name"} could have some meaning.
The same is for YAML:

```yaml
- app_version: 5.0.4
  description: phpMyAdmin is an mysql administration frontend
  repository:
    name: bitnami
    url: https://charts.bitnami.com/bitnami
  url: https://artifacthub.io/packages/helm/bitnami/phpmyadmin
  version: 8.1.0

```

Reply:
Good point. I hadn't thought of the JSON/YAML output formatters.